linalg/batched/batched.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2010-2024, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+
+#include "batched.hpp"
+#include "native.hpp"
+#include "gpu_blas.hpp"
+#include "magma.hpp"
+
+namespace mfem
+{
+
+BatchedLinAlg::BatchedLinAlg()
+{
+   backends[NATIVE].reset(new NativeBatchedLinAlg);
+
+   if (Device::Allows(~mfem::Backend::CPU_MASK))
```

Comment:
This check should be `Device::Allows(mfem::Backend::CUDA_MASK | mfem::Backend::HIP_MASK)` -- e.g. we don't want backends like `OMP` to use GPU_BLAS here.

Reply:
Fixed in 14c31edb4bfa22a484a77fa549fda8e28ee1a2e8.